### PR TITLE
Added the rest of the frameworks

### PR DESCRIPTION
**Pull request by muhakh**
Added the rest of the frameworks's approach in implementing router for their RESTful APIs.

### CODE REVIEW

File: manual/en-US/specification/chapters/routers-approaches.md [hunk 1]
```diff
@@ -58,5 +57,47 @@ For more info: https://laravel.com/docs/5.4/controllers#restful-resource-control
...
+Slim framework is a micro framework which is simple and good to build RESTful APIs.
+They build their routing within the app class as the developer specifies request by method, for example, if developer wants to add a route that handles only GET HTTP requests, it would be made as the following code.
+```
+$app = new \Slim\App();
+$app->get('/books/{id}', function ($request, $response, $args) {
+    // Show book identified by $args['id']
+});
+```
+Such approach is simple to understand, implement and apply in extensions, but it would take much work to specify a route for each method.
+## ASP<span>.</span>NET MVC framework
```
Comment (yvesh) [inline manual/en-US/specification/chapters/routers-approaches.md:91]: You know that this is for ASP.net, which requires another programming language than PHP (Like c#)? :P

**Comment by chrisdavenport** [comment]
Joomla Framework v2 router?

File: manual/en-US/specification/chapters/routers-approaches.md (hunk 1 above)
Comment (muhakh) [reply manual/en-US/specification/chapters/routers-approaches.md:91]: I also mentioned Django which is written in Python and Ruby on Rails which is written in Ruby :D

Comment (cokencorn) [reply manual/en-US/specification/chapters/routers-approaches.md:91]: Ruby is also pretty PHP like and I think it is worth considering Django's approach.

**Pull request merged**